### PR TITLE
FOGL-1856 - Tasks table should have a schedule_name column

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

**Comment by praveen-garg** [comment]
Upgrade is failing on postgres!

**Comment by amarendra-dianomic** [comment]
@praveen-garg foo task issue fixed. Pl check. Also, I am perfectly able to work on postgres via upgrade and did not face any issue at all. Please let me know your postgres error details.

**Comment by praveen-garg** [comment]
Now its failing on SQLite

File: scripts/plugins/storage/sqlite/init.sql [hunk 1]
```diff
@@ -506,7 +506,8 @@ CREATE TABLE foglamp.schedules (
 -- List of tasks
 CREATE TABLE foglamp.tasks (
              id           uuid                        NOT NULL,                          -- PK
-             process_name character varying(255)      NOT NULL,                          -- Name of the task
+             schedule_name character varying(255),                                       -- Name of the task
```
[the remark this answers is not in the document]
Comment (amarendra-dianomic) [reply scripts/plugins/storage/sqlite/init.sql]: For a fresh deployment, NOT NULL is ok for schedule_name column in tasks table but when we are upgrading from an existing installation, we have no way to fulfil this NOT NULL constraint. A default value of 'foo' was provided above for this purpose only which was removed after objections were raised. Also, to maintain consistency between a upgraded instance and a fresh instance, this NOT NULL constraint has been removed from both init.sql and upgrade script.

[the remark this answers is not in the document]
Comment (praveen-garg) [reply scripts/plugins/storage/sqlite/init.sql]: @MarkRiddoch  please advise on this.

**Pull request merged**